flink-python/pyflink/table/tests/test_descriptor.py

```diff
@@ -825,27 +825,27 @@ def test_field_in_string(self):
 
         properties = schema.to_properties()
         expected = {'schema.0.name': 'int_field',
-                    'schema.0.type': 'INT',
+                    'schema.0.data-type': 'INT',
                     'schema.1.name': 'long_field',
-                    'schema.1.type': 'BIGINT',
+                    'schema.1.data-type': 'BIGINT',
                     'schema.2.name': 'string_field',
-                    'schema.2.type': 'VARCHAR',
+                    'schema.2.data-type': 'VARCHAR(2147483647)',
```

Comment:
Why not `STRING`?

Reply:
STRING is an alias of `VARCHAR(2147483647)`, there is no `StringType`, so the displaying is `VARCHAR(2147483647)`.